aiida_zeopp/parsers/plain.py

```diff
@@ -215,6 +216,55 @@ def parse(cls, string):
         return res
 
 
+class PoresSizeDistParser(object):
+    @classmethod
+    def parse(cls, string):  # pylint: disable=too-many-locals
+        """
+        Parse zeo++ .psd format, using routines similar to other parsers to avoid additional pandas dependency
+
+        Parameters
+        ----------
+        string: str
+            string in psd format
+
+        Returns
+        -------
+        results: dict
+            dictionary of output values
+        """
+        lines = string.splitlines()
+        # remove empty lines
+        lines = [l for l in lines if l.strip()]
+
+        # find line where histogram data begins
+        header_line = 'Bin Count'
+        i = 0
+        for i, line in enumerate(lines):
+            if header_line in line:
+                break
+        else:
+            raise ValueError('Did not find header line in data')
+        # extract histogram data
+        bins, counts, cumulatives, derivatives = [], [], [], []
+        for line in lines[i + 1:]:
+            b, count, cumulative, derivative = line.split()
+            bins.append(float(b))
+            counts.append(float(count))
```

Comment:
Shouldn't a count be an integer?

Reply:
Thanks for noticing this, I'll change it to int.